.cloud-gov/manifest.yml

```diff
@@ -6,3 +6,4 @@
     memory: 512MB
     services:
       - all-sorns-db
+    command: bundle exec rake cf:on_first_instance db:migrate && bundle exec rails s -p $PORT -e $RAILS_ENV
```

Comment:
I'm surprised the command goes in the manifest and not the deploy script. That does make it work for the dev server too which is nice.

Try it out on the dev server, if it is acting up, then try adding the command to the end of the deploy script like 
`cf run-task "rails db:migrate"`

Reply:
this is what the docs suggested for ruby deploys. Either would work the nice thing here is it can pick up the correct instance.